lib/api.js

```diff
@@ -7,12 +7,16 @@ export function getAllPosts() {
 export function getPostBySlug(slug, type='') {
   if(type)
     return posts[type].find(post => post.slug === slug && !post.isSection)
-  else{
-    Object.entries(posts).forEach(([key, value]) => {
-      if(key!== 'docsTree')
-        return posts[key].find(post => post.slug === slug && !post.isSection)
-    })
-  }
+    else {
```

Comment:
can you do basic formatting improvement, indentation 😉

Reply:
yeahhh ,doing it